src/VisualStudio/Core/Next/project.json

```diff
@@ -1,21 +1,36 @@
 ﻿{
-    "dependencies": {
-        "Dev15BinariesForRoslyn": "5.0.0.0",
-        "Microsoft.VisualStudio.ImageCatalog": "15.0.25604-Preview4",
-        "NuGet.VisualStudio": {
-            "version": "3.3.0",
-            "suppressParent": "all"
-        },
-        "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": {
-            "version": "14.3.25407",
-            "suppressParent": "all"
-        },
-        "Microsoft.ServiceHub.Client": "0.13.78-alpha-gf674a884a9",
-        "Newtonsoft.Json": "8.0.3",
-        "RoslynDependencies.Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime": "15.0.25718-Preview5",
-        "RoslynDependencies.Microsoft.VisualStudio.Workspace": "14.0.983-pre-ge167e81694",
-        "System.Collections.Immutable": "1.2.0"
+  "dependencies": {
+    "Microsoft.VisualStudio.ImageCatalog": "15.0.25726-Preview5 ",
+    "NuGet.VisualStudio": {
+      "version": "3.3.0",
+      "suppressParent": "all"
     },
+    "Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime": {
+      "version": "14.3.25407",
```

Comment:
The only thing crazier than putting a version number in an assembly name is putting a version in an assembly name _and then updating it_.

I'll see about cleaning these up, but if we're just trying to pick up the COM interfaces it shouldn't matter.